app/views/journeys/capitalallowances/specialTaxSites/NewSiteClaimingAmountView.scala.html

```diff
@@ -32,7 +32,7 @@
 
 @(form: Form[_], mode: Mode, userType: UserType, taxYear: TaxYear, businessId: BusinessId, index: Int)(implicit request: Request[_], messages: Messages)
 
-@layout(pageTitle = title(form, messages("newSiteClaimingAmount.subHeading"))) {
+@layout(pageTitle = title(form, messages("capitalAllowance.claimingTheAllowance"))) {
```

Comment:
This should match the page's question i.e. `s"newSiteClaimingAmount.subHeading.$userType"`